initial clean up of slurm guide (#242)
northeastern-rc/rc-public-documentation: docs/source/slurmguide/slurmmonitoringandmanaging.md

(slurm-monitoring-and-managing)=

> # Monitoring and Managing Jobs

Reviewer: I feel like the overall organization of this page needs some work. There are a lot of headings that have only one sentence to describe the section/sections below.

Reviewer: It might be good to go through this page together.

> This section will explain the basics of checking the status of the cluster, checking for the details on the compute nodes, and checking on the running jobs you have on the cluster.

Reviewer: ```suggestion
You can use Slurm commands to check the status of the cluster, check details on the compute nodes, and check on the running jobs you have on the cluster. 
```

(cluster-and-node-states)=

> ## Cluster and Node States

Reviewer: These headings should match better with the description under Monitoring and Managing Jobs.

Reviewer: This heading also seems unnecessary.
